### PR TITLE
ddl: Fix the error format bug.

### PR DESCRIPTION
**Pull request by zimulala**

### CODE REVIEW

File: ddl/ddl_worker.go [hunk 1]
```diff
@@ -396,7 +396,8 @@ func toTError(err error) *terror.Error {
 	originErr := errors.Cause(err)
 	tErr, ok := originErr.(*terror.Error)
 	if ok {
-		return tErr
+		msg := tErr.Error()
```
Comment (shenli) [inline ddl/ddl_worker.go]: What's the different with the previous code?

Comment (zimulala) [inline ddl/ddl_worker.go]: origin: CREATE INDEX: index already exist %!s(MISSING)
now: CREATE INDEX: index already exist

**Comment by shenli** [comment]
LGTM

**Comment by shenli** [comment]
@tiancaiamao @coocood PTAL

**Comment by coocood** [comment]
The origin error should have an index name as a format argument, but the argument missing, this change doesn't print index name at all.

**Comment by c4pt0r** [comment]
Agree with @coocood 
This commit doesn't fix the original problem, but workaround it.

**Comment by zimulala** [comment]
The argument doesn't missing. It's only because my test argument is ` `. I'm sorry, I didn't express myself clearly. @coocood @c4pt0r

**Comment by coocood** [comment]
LGTM

**Pull request merged**